Gems/ImGui/Code/Include/ImGuiBus.h

```diff
@@ -10,9 +10,11 @@
 
 #include <AzCore/EBus/EBus.h>
 #include <AzCore/EBus/Event.h>
+#include <AzCore/Interface/Interface.h>
```

Comment:
Is the AZ::Interface used in the this bus header?
It doesn't seem like the include is needed here.

Reply:
Or did you want to make a type alias available to users for the AZ::Interface.

Such as
```c++
using ImGuiManagerInterface = AZ::Interface<IImGuiManager>;
```

Reply:
The type alias is probably unnecessary for the number of expected touchpoints. I'll remove the header instead to keep things lean and simple. Thanks!